charts/ecosystem/values.yaml

```diff
@@ -359,4 +359,22 @@ eventStreamsSecretName: "event-streams-token"
 #
 dssUrl: http://RELEASE_NAME-etcd:2379
 credsUrl: http://RELEASE_NAME-etcd:2379
-cpsUrl: http://RELEASE_NAME-etcd:2379
\ No newline at end of file
+cpsUrl: http://RELEASE_NAME-etcd:2379
+#
+#
+# cleanupMonitor represents the values required to configure a custom resource cleanup
+# monitor for use by the Galasa service, so that resources provisioned by managers that
+# are not publicly available in the Galasa open source offering can be cleaned up.
+#
+cleanupMonitor:
+  #
+  # The name of the stream in which custom resource cleanup providers can be located.
+  stream: ""
+  #
+  # A list of regex patterns to be used in identifying which resource cleanup providers
```

Comment:
Docs here will need more explanation of glob patterns...etc.

Reply:
Perhaps an example or two ?

Reply:
Addressed in https://github.com/galasa-dev/helm/pull/73/commits/6c4cebafb1f57664ea78dd16df11f913c4328d71